### DISCUSSION TITLE
CopyQ not calling external editor after last update

### DISCUSSION THREAD

**Discussion by technochitlin** [hluk/CopyQ#1883] [Q&A]
Hopefully this is the correct place to ask this. Since the last update, CopyQ (on Win10) stopped opening the external editor from the icon. I have this line in CopyQ.ini- 

editor=\"C:\\Program Files\\Notepad++\\notepad++.exe\" %1

Is this no longer correct?

**Comment by hluk** [comment]
Try replacing `\` with `/` or even `\\`. In this case, I think `\` might get special treatment (as an escape character).

**Reply by technochitlin** [reply]
editor=\"C:\\Program Files\\Notepad++\\notepad++.exe\" %1 is the actual entry in CopyQ.ini- I'm not sure how it got changed in my previous message, maybe the forum software "cleaned it up". (EDIT: well, yes it did. See the pic instead)

jpg: 
![image](https://user-images.githubusercontent.com/737753/150985934-14beaafc-008b-4a51-900f-80b36c86cff0.png)
